File: Api/Equipaments/Models/Equipment_Model_State_Hourly_Earnings.cs
using System;

namespace Equipments.Models
{
    public class Equipment_Model_State_Hourly_Earnings
    {
        
        public Guid equipment_model_id { get; set; }
        public Guid equipment_state_id { get; set; }

        public virtual Equipment_Model equipment_model { get; set; }

        public virtual Equipment_State equipment_state { get; set; }
        public double value { get; set; }
    }
}

File: project/Forest-operation-project/src/main/java/com/app/project/domain/EquipmentModelStateHourlyEarnings.java
package com.app.project.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.math.BigDecimal;
import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Entity
@Table(name = "equipment_model_state_hourly_earning")
public class EquipmentModelStateHourlyEarnings {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;

    private BigDecimal value;

    @ManyToOne
    @JoinColumn(name = "equipment_model_id", referencedColumnName = "id")
    private EquipmentModel equipmentModel;

    @ManyToOne
    @JoinColumn(name = "equipment_state_id", referencedColumnName = "id")
    private EquipmentState equipmentState;
}
